mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -573,6 +580,22 @@ properties:
     name: 'disableTcpConnections'
     description: |
       Disables support for plain TCP connections in the workstation. By default the service supports TCP connections via a websocket relay. Setting this option to true disables that relay, which prevents the usage of services that require plain tcp connections, such as ssh. When enabled, all communication must occur over https or wss.
+  - !ruby/object:Api::Type::Array
```

Comment:
It's this top-level field `required` as well since all inner fields are `required`. In that case this field needs to be updated in all existing tests for this resource.

Reply:
Hi @hao-nan-li, `allowed_ports` it self is not required, however, if defined `first` and `last` are required inside the `allowed_ports` argument. Please let me know if i configured this correctly!